test/test_datasets.py

```diff
@@ -2206,6 +2206,52 @@ def inject_fake_data(self, tmpdir: str, config):
         return len(sampled_classes * n_samples_per_class)
 
 
+class FVGCAircraftTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.FVGCAircraft
+    FEATURE_TYPES = (PIL.Image.Image, int)
+
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(split=("train", "val", "trainval", "test"))
+
+    def inject_fake_data(self, tmpdir: str, config):
+        split = config["split"]
+        root_folder = pathlib.Path(tmpdir) / "fgvc-aircraft-2013b"
+        data_folder = root_folder / "data"
+
+        num_images_per_class = 5
+        variants = ["707-320", "Hawk T1", "Tornado"]
+        n_samples_per_class = 4 if split == "trainval" else 2
```

Comment:
Nit, for our naming style consistency (although I personally prefer `n_` as well):

```suggestion
        num_samples_per_class = 4 if split == "trainval" else 2
```